packages/client/src/api/api.service.ts

```diff
@@ -63,8 +70,15 @@ export class ApiService {
    * @deprecated The method should not be used - Use markMessageAs instead.
    */
   async markMessageAsRead(messageId: string | string[]): Promise<any> {
+    const messageIdString =
+      !messageId || typeof messageId === 'object'
+        ? Array.isArray(messageId)
+          ? messageId.toString()
+          : ''
+        : messageId.toString();
```

Comment:
```suggestion
    const messageIdString = messageId ? messageId.toString() : '';
```

I think this can be simplified to this right?

Reply:
Wasn't sure if the DS analyzer would pass for the object type data, so added a condition to take care of that scenario also.
Will make the change